listen/__init__.py

```diff
@@ -1,9 +1,10 @@
-__version__ = "0.2.0"
-__author__ = 'Byron Vanstien'
+__version__ = "0.3.0"
+__author__ = 'Byron Vanstien, Yarn'
 __license__ = 'MIT'
 __copyright__ = 'Copyright 2016-2017 Byron Vanstien'
 __title__ = 'listen'
 
 from listen.client import Client # noqa
 from listen.errors import KanaError, ListenError # noqa
 from listen.objects import User, Song # noqa
+from . import message
```

Comment:
I'd prefer if we stuck to absolute imports. They are the "recommended" way of doing things